### DISCUSSION TITLE
What is used to build winutil?

### DISCUSSION THREAD

**Discussion by t-lit** [ChrisTitusTech/winutil#1698] [Q&A]
I'm looking to build something similar to winutil and was wondering what was used to build the GUI.

Could you point in me in the right direction to creating something like this?

**Comment by ModernTTY** [comment]
Winutil is just a regular PowerShell script that uses WPF for the GUI.

**Reply by t-lit** [reply]
How is the GUI edited? Is just manually editing the xml?

**Reply by ModernTTY** [reply]
Some yes, some is compiled from JSON files.

**Reply by eraldinho** [reply]
Hello, is there a software with which I can "see what i get" when editing the xml?
